src/error/GraphQLError.js

```diff
@@ -1,11 +1,4 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- *
- * @flow strict
- */
```

Comment:
I looked at the source code for a couple of other projects under LF (Node.js, ESLint, esprima) and they just just have LICENSE at the root of the project and no license headers inside the source code.

Those headers add some inconvenience in the development process e.g. new contributors forget to add them, old PRs contains an old version of the license text, it's harder to diff and revert old commits, etc.

Plus they effectively double up the size of NPM package:
```
npm notice package size:  483.3 kB
npm notice unpacked size: 3.6 MB
```
VS
```
npm notice package size:  237.5 kB
npm notice unpacked size: 1.7 MB
```

Reply:
> Plus they effectively double up the size of NPM package:

I mad mistake during measurement, in reality, it was around ~100 kB difference unpacked.